proxy/src/transparency/client.rs

```diff
@@ -169,12 +169,32 @@ where
     }
 
     fn call(&mut self, req: Self::Request) -> Self::Future {
+        use http::header::CONTENT_LENGTH;
+
         match self.inner {
             ClientServiceInner::Http1(ref h1) => {
+                // As of hyper 0.11.x, a set body implies a body must be sent.
```

Comment:
Let's preface this comment with a reference to the RFC, something like:

// https://tools.ietf.org/html/rfc7230#section-3.3 says "The presence of a message body in a request is signaled by a Content-Length or Transfer-Encoding header field."